File: cfgs/java_cfgs_foo/COUNT_MINIMUM_NUMBER_SUBSETS_SUBSEQUENCES_CONSECUTIVE_NUMBERS.java
Nodes
1;special;;;
2;statement;fun_1(foo_1);14;
3;statement;int count = 1;15;
4;statement;int i = 0;16;
5;conditional;foo_1 < (foo_2 - 1);17;
6;conditional;(foo_1[foo_2] + 1) != foo_3[foo_4 + 1];19;
7;statement;count++;19;
8;statement;i++;18;
9;statement;return foo_1;21;
10;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;9;
5;7;8;
6;8;
7;5;
8;10;
9;

File: cfgs/pythoncfgs/COUNT_MINIMUM_NUMBER_SUBSETS_SUBSEQUENCES_CONSECUTIVE_NUMBERS.py
Nodes
1;statement;foo_1 = fun_1(foo_2);8;
2;statement;foo_1 = 1;9;
3;conditional;foo < foo;10;
4;statement;return foo_1;13;
5;conditional;foo_1[foo_2] + 1 != foo_3[foo_4 + 1];11;
6;statement;foo_1 = foo_2 + 1;12;
7;exit;;
Edges
0;2;
1;3;
2;4;5;
3;7;
4;3;6;
5;3;
6;